lib/fog/cloudstack/requests/compute/disable_static_nat.rb

```diff
@@ -19,6 +19,32 @@ def disable_static_nat(*args)
         end
       end
 
+      class Mock
+        def disable_static_nat(*args)
+          ip_address_id = args[0].is_a?(Hash) ? args[0]['ipaddressid'] : args[0]
+
+          address = self.data[:public_ip_addresses][ip_address_id]
+
+          unless address
+            raise Fog::Compute::Cloudstack::BadRequest.new(
+"Unable to execute API command disablestaticnat due to invalid value. \
+Invalid parameter ipaddressid value=#{ip_address_id} due to incorrect long value format, \
+or entity does not exist or due to incorrect parameter annotation for the field in api cmd class.")
+          end
+
+          if address['virtualmachineid'].nil?
+            raise Fog::Compute::Cloudstack::BadRequest.new(
+"Specified IP address id is not associated with any vm Id")
+          end
+
+          address.merge!(
+            'virtualmachineid'          => nil,
+            'virtualmachinname'         => nil,
+            'virtualmachinedisplayname' => nil,
```

Comment:
Could you remove this trailing comma? It seems to be causing issues in the build for 1.8.7. Thanks!